.travis.yml

```diff
@@ -1,69 +1,70 @@
 language: python
+
 python:
   - "2.7"
   - "3.5"
 
-# Setup numpy + scipy using miniconda
-# See http://conda.pydata.org/docs/travis.html
-install:
-  - sudo apt-get update
-
-  # Install FFTW for pyFFTW
-  - sudo apt-get install libfftw3-dev
+sudo: false
 
-  # Install LaTeX to render the math formulas in the docs
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "3.5" && "$TRAVIS_BRANCH" == "master" ]]; then
-      sudo apt-get install texlive-latex-base texlive-latex-recommended texlive-latex-extra dvipng;
-    fi
+addons:
+    apt:
+        packages:
+            libfftw3-dev
+            texlive-latex-base
+            texlive-latex-recommended
+            texlive-latex-extra
+            dvipng
 
-  # Install miniconda according to Python version of the build (saves downloading if versions match)
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
-    else
-      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-    fi
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
-  - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
-  # Useful for debugging any issues with conda
-  - conda info -a
+# Enable cache for $HOME/.cache/pip (wheels and such)
+cache: pip
 
-  # Install dependencies and enter test environment
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy future nomkl sphinx sphinx_rtd_theme
-  - source activate test-environment
+# Setup numpy + scipy using miniconda
```

Comment:
From here down no changes except `py.test` -> `pytest`. Just messed up by the GH diff.